src/delivery/http/mod.rs

```diff
@@ -189,7 +189,7 @@ impl APIClient {
     pub fn get(&self, path: &str) -> Result<HyperResponse, HttpError> {
         let url = self.api_url(path);
         let mut client = hyper::Client::new();
-        let req = client.get(url.as_slice());
+        let req = client.get(url.as_ref());
```

Comment:
Due to generics, the Deref coercions don't appropriately kick in. At least, I'm pretty sure that's it :)